beta-src/src/models/dummy.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react";
+import BoardClass from "./BoardClass";
+import OrderClass from "./OrderClass";
+
+import getData from "./mockAPI";
+
+/**
+   * list of order generation methods
+   *  getMoveChoices();
+      getSupportHoldChoices();
+      getSupportMoveToChoices();
+      getConvoyToChoices();
+
+      These two methods still need to be refactored.
+      getSupportMoveFromChoices();
+      getConvoyFromChoices();
+   */
+
+const defaultOptions = [
+  { name: "Hold" },
+  { name: "Move" },
+  { name: "Support Hold" },
+  { name: "Support Move" },
+];
+
+interface WDOrderInterfaceProps {
+  userOrder: OrderClass;
+}
+
+const OrderInterface: React.FC<WDOrderInterfaceProps> = function ({
+  userOrder,
+}): React.ReactElement {
+  /**
+   * Bottom three will be used for order submission.
+   */
+  const [currOrder, setCurrOrder] = React.useState(userOrder);
```

Comment:
steps:

1. remove `React.`
2. hover over `useState` and click quick fix
3. choose import useState from react

then remove all over React.

Reply:
@nebevets @hyukcodazen actually I think we should keep dummy.tsx and the testing stuff until we start to integrate the logic into the real app.

Reply:
@bryancodazen Jason created a new ticket for implementing new test suite for refactored code. But I'm fine keeping dummy.tsx for testing purpose as well.

Reply:
@hyukcodazen the test suite is different and we will get there, but keeping this for now this gives us examples of usage right off the bat.